services/secret-service/server/src/main/resources/secrets.json

```diff
@@ -32,5 +32,11 @@
     "type": "DROPBOX",
     "secretId": "dropbox-cred",
     "accessToken": ""
+  },
+  {
+    "type": "FTP",
```

Comment:
Remove passwords and specific user ids. Make them generic as possible

Reply:
This is the generic username and password for testing. The website https://dlptest.com/ftp-test/ offers an FTP server for testing purposes. The files remain for 30 mins from the time of upload. I left for testing purposes. Since it's publicly available should I remove it?